### DISCUSSION TITLE
Train epoch=20,but in the snapshot : only have e1-e19

### DISCUSSION THREAD

**Discussion by smile-hahah** [STVIR/pysot#188] [General] [answered]
When I was training this project  two weeks ago,in the snapshot folder,here will generate 20 files(e1-e20).But now,I find in this folder only have 19 files(e1-e19).Both train epoch are the same(train epoch=20).But I don't know why？Thanks for your reply.

**Comment by ZhiyuanChen** [comment]


**Reply by huchenjie339** [reply]
@ZhiyuanChen 我最近训练也遇到了这个问题，总共训练了20epoch，理应在snapshot文件夹下保存20个网络模型才对，但是只保存了19个epoch的模型，这是什么原因呢，希望得到你的回复和解答。
![image](https://user-images.githubusercontent.com/62690860/103532901-13a57100-4ec7-11eb-99d1-36ce21b6513f.png)

**Comment by ZhiyuanChen** [comment, marked as answer]
There is no point to save the checkpoint of epoch 0, as it is a bunch of random numbers.
